EXTERNAL/csparse/CMakeLists.txt

```diff
@@ -60,6 +60,9 @@ ADD_LIBRARY(csparse ${G2O_LGPL_LIB_TYPE}
 )
 
 SET_TARGET_PROPERTIES(csparse PROPERTIES OUTPUT_NAME ${LIB_PREFIX}ext_csparse)
+if (APPLE)
+  set_target_properties(csparse PROPERTIES INSTALL_NAME_DIR "${CMAKE_INSTALL_PREFIX}/lib")
```

Comment:
nit: you may want to match the casing of the other calls to SET_TARGET_PROPERTIES for consistency

Reply:
Thanks for the comment.

The cmake files in general are fairly inconsistent in their use of cases (e.g., CMakeLists.txt for g2o_viewer) and indentation (e.g., g2o_cli).

I checked on the cmake coding styles and the format used is rather out of date - e.g., the cmake developers were recommending in 2012 that all the commands should be in lower case:

https://github.com/Kitware/CMake/commit/77543bd

Perhaps we should apply this change and make sure everything is consistent?